src/main/resources/templates/map-location.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout/base}">
+
+<head>
+    <title>Calculate Journey Variable Payments - GOV.UK</title>
+</head>
+
+<body>
+
+<div class="govuk-width-container" layout:fragment="content">
+    <a href="/dashboard" class="govuk-back-link">Back</a>
+
+    <form th:action="@{/map-location}" th:object="${form}" method="post">
+        <div class="govuk-form-group" th:classappend="${#fields.hasErrors('locationName')} ? 'govuk-form-group--error'">
+            <h1 class="govuk-label-wrapper"><label class="govuk-label govuk-label--l" for="event-name">
+                Map location
+            </label>
+            </h1>
+            <div id="map-location-name-hint" class="govuk-hint">
+                Please add a Schedule 34 location name for this NOMIS agency ID.
+            </div>
+            <div id="agency-id-hint" class="govuk-hint">
+                <span class="db govuk-!-font-weight-bold">Pick up location</span>
+                <p th:text="*{agencyId}"></p>
+                <input type="hidden" th:field="*{agencyId}"/>
```

Comment:
Should/could this be a session attribute instead?

Reply:
No i think hidden is good